core/src/main/java/org/apache/cxf/helpers/FileUtils.java

```diff
@@ -293,67 +290,6 @@ public static File createTempFile(String prefix, String suffix, File parentDir,
         return result;
     }
 
-    public static String getStringFromFile(File location) {
```

Comment:
I'm not sure if it's wise to be removing public methods for a minor patch, maybe we should wait until we move to 3.4.0-SNAPSHOT?

Reply:
ok

Reply:
@coheigea WDYT?